src/frontend/components/DashboardVideoLive/index.tsx

```diff
@@ -1,48 +1,40 @@
-import ClipboardJS from 'clipboard';
 import { Box, Heading, Text } from 'grommet';
-import React, { useEffect } from 'react';
+import React, { lazy, useEffect } from 'react';
 import { defineMessages, FormattedMessage } from 'react-intl';
-import ReactTooltip from 'react-tooltip';
-import styled from 'styled-components';
 
 import { appData } from '../../data/appData';
 import { useVideo } from '../../data/stores/useVideo';
 import { API_ENDPOINT } from '../../settings';
 import { modelName } from '../../types/models';
-import { Video, liveState } from '../../types/tracks';
+import { Video, liveState, LiveModeType } from '../../types/tracks';
 import { report } from '../../utils/errors/report';
 import { CHAT_ROUTE } from '../Chat/route';
 import { PLAYER_ROUTE } from '../routes';
 import { DashboardVideoLiveStartButton } from '../DashboardVideoLiveStartButton';
 import { DashboardVideoLiveStopButton } from '../DashboardVideoLiveStopButton';
 import { DashboardButtonWithLink } from '../DashboardPaneButtons';
 
+const DashboardVideoLiveRaw = lazy(() => import('../DashboardVideoLiveRaw'));
+const DashboardVideoLiveJitsi = lazy(
+  () => import('../DashboardVideoLiveJitsi'),
+);
+
 const messages = defineMessages({
-  copied: {
-    defaultMessage: 'Copied!',
-    description: 'Message displayed when endpoints info are copied.',
-    id: 'components.DashboardVideoLive.copied',
-  },
-  title: {
+  raw: {
     defaultMessage: 'Streaming link',
     description: 'DashboardVideoLive main title.',
     id: 'components.DashboardVideoLive.title',
```

Comment:
I really don't know if there's a mistake, but usually, id and key matches, no ?
```suggestion
    id: 'components.DashboardVideoLive.raw',
```

Reply:
Yes it could, I will change it